addon/decorators/route-pagination.ts

```diff
@@ -28,7 +28,7 @@ export default function routePagination<T extends ConcreteSubclass<any>>(RouteSu
          * Get the controller params
```

Comment:
same here, these `addon/decorators/...` files should be removed completely, in favor of only having the `addon/mixins/..` typescript mixin versions.